app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
+//= link_directory ../stylesheets .css
+//= link_tree ../builds
```

Comment:
- [ ] Your assets config for this project has issues. I received a double link error when setting up the project because you have multiple CSS output files in stylesheets and builds. Also you did not declare Javascript as an asset for this project so it could not compile the javascript include tag in the views. To fix all these issues this is what you should have in this file and your app now works as expected.

  ```
  //= link_tree ../images
  //= link_directory ../stylesheets .css
  //= link application.js
  ```

  ![image](https://github.com/Jawad-Nazari/hello-rails-react/assets/81468675/6db004c4-37e3-4daa-8168-39707c2d93ae)

Reply:
Hi @julie-ify ,
Thanks for your time and for reviewing my code. I appricitae you for provided solution. 🙏